<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1289</width>
    <height>1307</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QSlider" name="Hw1_slider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="Hw1_spin">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="Hw2_spin">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="Hw2_slider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Hw3_slider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="Hw3_spin">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>440</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="Hw4_spin">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>580</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="Hw4_slider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>600</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="Hw5_spin">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>720</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="Hw5_slider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>740</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Hw6_slider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>880</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="Hw6_spin">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>860</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="Hw7_spin">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>1000</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="Hw7_slider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>1020</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Hw8_slider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>1160</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="Hw8_spin">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>1140</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="Homework_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Homework 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="Homework_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Homework 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="Homewor_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Homework 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="Homework_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>550</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Homework 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="Homework_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>690</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Homework 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="Homework_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>830</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Homework 6</string>
    </property>
   </widget>
   <widget class="QLabel" name="Homework_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>970</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Homework 7</string>
    </property>
   </widget>
   <widget class="QLabel" name="Homework_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>1110</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Homework 8</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>800</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>940</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>1080</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Midterm_1">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>130</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Midterm 1</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Mt1_spin">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>160</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="Mt1_slider">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>180</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>20</width>
      <height>1241</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="Mt2_spin">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>300</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>380</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Mt2_slider">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>320</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Midterm_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>270</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Midterm 2</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Final_spin">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>440</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="Final">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>410</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Final</string>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>520</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Final_slider">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>460</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="Schema_Box">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>580</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Scheme A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Scheme B</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>660</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Final_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>830</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Overall Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Homework_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>541</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pic10B Grade Calculator</string>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>700</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calculate Grade</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Score_display">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>860</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1289</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGrade_Calculator">
    <property name="title">
     <string>Grade Calculator</string>
    </property>
   </widget>
   <addaction name="menuGrade_Calculator"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Hw1_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hw1_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hw1_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hw1_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>509</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hw2_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hw2_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hw2_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hw2_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hw3_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hw3_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hw3_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hw3_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hw4_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hw4_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>636</y>
    </hint>
    <hint type="destinationlabel">
     <x>509</x>
     <y>622</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hw4_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hw4_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>661</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>646</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hw5_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hw5_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>780</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>763</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hw5_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hw5_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>794</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>784</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hw6_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hw6_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>920</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>897</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hw6_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hw6_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>925</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>927</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hw7_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hw7_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>1071</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>1055</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hw7_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hw7_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>1056</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>1049</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hw8_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hw8_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>1200</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>1188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hw8_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Hw8_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>1205</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>1204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Mt1_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Mt1_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1169</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>1015</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Mt1_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Mt1_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>982</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>1172</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Mt2_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Mt2_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1045</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>1174</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Mt2_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Mt2_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1166</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>1083</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Final_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Final_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1154</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>1053</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Final_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Final_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1022</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>1184</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
